pkg/kv/local/store.go

```diff
@@ -43,6 +43,9 @@ func (s *Store) Get(ctx context.Context, partitionKey, key []byte) (*kv.ValueWit
 
 	var value []byte
 	err := s.db.View(func(txn *badger.Txn) error {
+		if ctx.Err() != nil {
```

Comment:
all the operations on badger transaction support passing context ready.

Reply:
Unfortunately it isn't.

Reply:
Don't think we should handle the context cancellation here - as part of long iteration it is acceptable, but single get I think it will just bloat the code.

Reply:
I agree - but we have acceptance tests we must pass. I think it's reasonable to expect a cancellation error from the `store` when the context was cancelled before the operation. If we remove it from here we can't expect it from other stores.

Reply:
We can't except context cancellation from all our underlaying implementation - for example mem implementation doesn't use context.
Not all functionallity should check and return ctx error.